.vscode/tasks.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Could you rename the commit with something more explicit, like `Add VS Code tasks`